controller/ProviderAdmin.php

```diff
@@ -41,4 +44,33 @@ public function getClassService()
     {
         return $this->getServiceLocator()->get(RdfLtiProviderRepository::class);
     }
+
+    protected function getExtraValidationRules(): array
+    {
+        return $this->getValidationFactory()->createFormValidators($this->getLtiVersion());
+    }
+
+    private function getLtiVersion(): string
```

Comment:
If I do not select `LTI version` in the form (what should be possible, cause we are going to hide those fields with feature flag), I got the following error:

```
Fatal error
: Uncaught TypeError: Return value of oat\taoLti\controller\ProviderAdmin::getLtiVersion() must be of the type string, null returned in /var/www/html/taoLti/controller/ProviderAdmin.php:61 Stack trace: #0 /var/www/html/taoLti/controller/ProviderAdmin.php(50): oat\taoLti\controller\ProviderAdmin->getLtiVersion() #1 /var/www/html/tao/actions/class.RdfController.php(590): oat\taoLti\controller\ProviderAdmin->getExtraValidationRules() #2 [internal function]: tao_actions_RdfController->editInstance() #3 /var/www/html/tao/models/classes/routing/ActionEnforcer.php(243): call_user_func_array(Array, Array) #4 /var/www/html/tao/models/classes/routing/ActionEnforcer.php(198): oat\tao\model\routing\ActionEnforcer->resolve(Object(GuzzleHttp\Psr7\ServerRequest)) #5 /var/www/html/tao/models/classes/routing/ActionEnforcer.php(175): oat\tao\model\routing\ActionEnforcer->execute() #6 /var/www/html/tao/models/classes/routing/TaoFrontController.php(101): oat\tao\model\routing\ActionEnforcer->__invoke(Object(GuzzleHttp\Psr7\ServerR in
/var/www/html/taoLti/controller/ProviderAdmin.php
on line
61

```